src/Tools/ConfigurationSchemaGenerator/ConfigSchemaEmitter.cs

```diff
@@ -268,98 +430,58 @@ private static void GenerateDocCommentsProperties(JsonObject propertyNode, strin
         }
     }
 
-    private void GenerateTypeDocCommentsProperties(JsonObject? currentNode, TypeSpec type)
+    internal static string FormatDescription(XElement element)
```

Comment:
This algorithm is a bit surprising. We inject `<br/><br/>` in ConvertBlankLines and then here turn them to `\n\n\n\n`, and then `\n\n`.

Can you add some comments here to explain it?

Reply:
I've pushed a commit to explain. Tests exist for all whitespace replacements around `<br/>`.